zebrad/src/components/tracing/component.rs

```diff
@@ -266,7 +270,23 @@ impl Tracing {
             info!("activated progress bar");
 
             if config.log_file.is_some() {
-                eprintln!("waiting for initial progress reports...");
+                // Clear screen
```

Comment:
I don't think this logo needs to depend on the progress bar feature. We could activate it whenever the log file is enabled. This would also fix the "unused variable network" warnings.

Reply:
Good point. Thinking about it, I think it makes sense to print the logo if it's a terminal. It seems wasteful sending it to a log. And that's what zcashd does too. I did that on da446250